packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -17,7 +17,7 @@ import deviceInfoModule from 'react-native-device-info'
 import Account from 'src/account/Account'
 import FiatExchange from 'src/account/FiatExchange'
 import GoldEducation from 'src/account/GoldEducation'
-import { default as InviteScreen } from 'src/account/Invite'
+// import { default as InviteScreen } from 'src/account/Invite'
```

Comment:
remove?

Reply:
Well, I did not want to remove these references, cause I've left `Invite` screen commented down below until we can redesign it as well. So, I would either remove them all or left all the references commented. WDYT?

Reply:
I've decided to remove it for now completely